### ISSUE TITLE
Remove chapters

### ISSUE THREAD

**Issue by annevk** [w3c/webvtt#344]
Per https://github.com/whatwg/html/pull/2481 they are not implemented and at least Chromium has no plans.

Unless more information comes forth they'll most likely be removed from the HTML Standard and should be removed from WebVTT as well.

**Comment by silviapfeiffer** [comment]
I think there's a natural cull going to happen when we move this spec to CR and further where not implemented features will need to be removed. @benjaminschaaf can make sure to have a test category for this.

**Comment by gkatsev** [comment]
A lot of web-based media players like Video.js and JW Player do support chapters tracks. Not sure whether and how it should affect but it's another data point.

**Comment by annevk** [comment]
@gkatsev that's very interesting for https://github.com/whatwg/html/issues/2507. In particular, whether you rely on the browser returning "chapter" from `<track>.kind`.

**Comment by gkatsev** [comment]
I can't speak for the other players, but in Video.js we do look for `<track>.kind` to equal `chapters`.

**Comment by silviapfeiffer** [comment]
Yeah, I forgot about the js video players. They all rely on the track kind. We need to make sure not to Cull features only because browsers don't support them. Also, existing the "kind" value is actually supported in all browsers.

**Comment by annevk** [comment]
Browsers shouldn't have reflected "chapter" before they added chapter support, but now that they did I agree that we should probably keep doing that.

**Comment by silviapfeiffer** [comment]
It's a semantic interpretation of the meaning of the track - even if there's no other implementation in the browsers, it's good to know what the track is meant to support. This includes 'descriptions' and 'metadata' track kinds as well. This is comparable to other semantic markup, e.g., a 'header' or 'footer' element which is in preference over a mere 'div' element. So I don't see a need to remove them.

**Comment by annevk** [comment]
`header` and `footer` elements have implementation requirements attached to them, such as rendering and how they are exposed to the accessibility tree.

**Comment by silviapfeiffer** [comment]
For 'chapter' tracks, a rendering requirement could be added but they should definitely be exposed in the accessibility tree so to make it easier for users to access semantic sections of a video.
For rendering, a mere statement that chapters should be visible in the video controls would be sufficient. Check out the JWPlayer implementation of this https://support.jwplayer.com/customer/portal/articles/1407454-adding-chapter-markers 

Might be something to consider adding to the HTML spec?

**Comment by annevk** [comment]
Yeah maybe, if we can find implementers that would implement it.

**Comment by dwsinger** [comment]
It's worth realizing that noticing that something is a chapter track and consequently NOT displaying it as if it were captions, IS implemented.

**Comment by silviapfeiffer** [comment]
I think we can close this issue - we should not remove chapter tracks at this stage IMO.

**Comment by zcorpan** [comment]
We should clarify that they can be used for JS consumption, but remove user agent requirements on exposing chapters in user agents. Basically align with what is implemented.

This will mostly affect HTML, but I think some changes will be necessary in WebVTT as well, if nothing else because some HTML terms that WebVTT links to will likely go away.

**Comment by silviapfeiffer** [comment]
The only reason for removing the implementation requirement would be to mirror implemented reality. There are huge accessibility and usability advantages to having chapters, so ultimately, it would be good to continue to encourage browser implementations. So, I think a WD should keep it, but a Rec shouldn't.

**Comment by AliceWonderMiscreations** [comment]
Chapter support should not be removed.
I am creating a new player now - for audio - that implements captions, subtitles, and chapters. Odd that browsers don't implement at least captions for audio, but they don't.

Anyway chapters are very useful for navigating longer audios.

Please don't let the laziness of browser devs with respect to chapters remove something that is very useful for custom html5 interfaces that provide what browsers don't.

Also WebVTT chapters can be very useful for non-browser implementations, like media in MKV containers.
